src/assets/images/close-cross.svg

```diff
@@ -1,4 +1,4 @@
 <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M0.393823 2.29537C-0.131274 1.77027 -0.131274 0.91892 0.393823 0.393823C0.91892 -0.131274 1.77027 -0.131274 2.29537 0.393823L15.6062 13.7046C16.1313 14.2297 16.1313 15.0811 15.6062 15.6062C15.0811 16.1313 14.2297 16.1313 13.7046 15.6062L0.393823 2.29537Z" fill="#9492C4"/>
-<path d="M2.29537 15.6062C1.77027 16.1313 0.918922 16.1313 0.393825 15.6062C-0.131272 15.0811 -0.131272 14.2297 0.393825 13.7046L13.7046 0.393824C14.2297 -0.131273 15.0811 -0.131273 15.6062 0.393824C16.1313 0.918921 16.1313 1.77027 15.6062 2.29537L2.29537 15.6062Z" fill="#9492C4"/>
+<path d="M0.393823 2.29537C-0.131274 1.77027 -0.131274 0.91892 0.393823 0.393823C0.91892 -0.131274 1.77027 -0.131274 2.29537 0.393823L15.6062 13.7046C16.1313 14.2297 16.1313 15.0811 15.6062 15.6062C15.0811 16.1313 14.2297 16.1313 13.7046 15.6062L0.393823 2.29537Z" fill="currentColor"/>
+<path d="M2.29537 15.6062C1.77027 16.1313 0.918922 16.1313 0.393825 15.6062C-0.131272 15.0811 -0.131272 14.2297 0.393825 13.7046L13.7046 0.393824C14.2297 -0.131273 15.0811 -0.131273 15.6062 0.393824C16.1313 0.918921 16.1313 1.77027 15.6062 2.29537L2.29537 15.6062Z" fill="currentColor"/>
```

Comment:
using `currentColor` will make the SVG use whatever `color` prop the parent uses. We should actually have that on all of our SVGs.

Reply:
clever